vpr/src/place/move_transactions.cpp

```diff
@@ -2,6 +2,18 @@
 
 #include "globals.h"
 #include "place_util.h"
+#include "vtr_assert.h"
+
+t_pl_blocks_to_be_moved::t_pl_blocks_to_be_moved(size_t max_blocks){
+        moved_blocks.reserve(max_blocks);
+        moved_blocks.resize(0);
```

Comment:
is calling resize() necessary?

Reply:
No, it is not necessary, the vector default constructor will set size to 0.

Reply:
> No, it is not necessary, the vector default constructor will set size to 0.

So I think it is better to remove it